docs/changelog.rst

```diff
@@ -123,6 +129,14 @@ Cleanups
 - Catch errors when loading :confval:`logging/config_file`.
   (Fixes: :issue:`1320`)
 
+Audio
+-----
+
+- **Breaking:** The audio scanner now returns ISO-8601 formatted strings
```

Comment:
I don't think Tunein is using the Scanner instance it creates any more. Looks like it's in a bit of a state after that last PRs I merged. So no, no issue there.